docs/faq.md

```diff
@@ -43,6 +44,28 @@ Example crontab entry that starts the processor every minute:
 
 `* * * * * cd /piwik && ./console queuedtracking:process >/dev/null 2>&1`
 
+Example Supervisor entry that will starts 16 processor/worker with 10 loop cycle times and auto restart:
```

Comment:
Just some slight typo corrections:
```suggestion
Example Supervisor entry that will start 16 processors/workers with 10 loop cycle times and auto restart:
```

Reply:
honestly, i'm bad at writing documentation, especially in english 😄✌️. i'll fix it..